cmd/fs-v1-multipart.go

```diff
@@ -799,6 +800,19 @@ func (fs fsObjects) CompleteMultipartUpload(bucket string, object string, upload
 				PartETag:   part.ETag,
 			})
 		}
+		if partSize == -1 {
```

Comment:
A comment like, you set the partsize value to first available size and you would be using that variable to make sure that all the remaining parts are of same size.

Reply:
I have added a comment